src/components/alert/alert/alert.stories.ts

```diff
@@ -166,6 +166,7 @@ const defaultArgs: Args = {
   target: undefined,
   rel: undefined,
   'accessibility-label': undefined,
+  animation: animation.options[0],
```

Comment:
hmm with our new default set to none, it's difficult to see the animation in the storybook.

Should, maybe as we have the prop now, the standard still be "open" respect. "all" for notification? Not sure :/